src/include/cursor.h

```diff
@@ -240,6 +240,10 @@ struct __wt_cursor_btree {
     /* Check that cursor next/prev never returns keys out-of-order. */
     WT_ITEM *lastkey, _lastkey;
     uint64_t lastrecno;
+
+    WT_REF *lastref;    /* The page where the last key is */
```

Comment:
Can you add a comment explaining what debug information that the following provides to find out the OOO issue would help.